test/compose/line-completion/in

```diff
@@ -1,4 +1,4 @@
             w111111
 
     w222222
-                          
+                          w2
```

Comment:
This test case also passes without your change. I'm not sure what your change is supposed to do?
(Maybe it's better to make a separate test case because this one remains valid)

Reply:
The original test case will match both `w222222` and `w111111`, but the current line completer has the prefix filter functionality which is not covered in the original test case. So I just amend the test case to cover this.

Reply:
> BTW, if we complete the line with prefix w3, the previous implementation will also provide both w222222 and w111111 as candidates.

Sorry I can't reproduce - when I replace `w2` with `w3` I correctly get no completions on `<c-x>l` on master - can you provide a test case?

Reply:
It's weird, I can reproduce in my side with `5696ed02`.  For the test case, you can just use this amended test case which I add `w2` as the prefix, in the `cmd` you can see if you trigger line completion, you will get two candidates which are `w222222` and `w111111` w/o this patch,  actually, I expect only `w111111` should show.